.vsts-dotnet.yml

```diff
@@ -11,6 +11,10 @@ trigger:
 #   SkipApplyOptimizationData: false
 
 variables:
+  - ${{ if startsWith(variables['Build.SourceBranch'], 'refs/heads/exp/')) }}:
+    SourceBranch: master
```

Comment:
Why is this format different from the lines just below?

Reply:
Changed it to match the format below